package.json

```diff
@@ -30,6 +30,7 @@
         "plugins/github-pages",
         "plugins/github-pages/example",
         "plugins/path-to-regexp",
+        "plugins/koa",
```

Comment:
Let's add the example to the `workspaces` field as well. So that yarn manages the example's dependencies as well